include/irr/asset/IAssetWriter.h

```diff
@@ -42,6 +57,8 @@ class IAssetWriter : public virtual core::IReferenceCounted
         size_t encryptionKeyLen;
         const uint8_t* encryptionKey;
         const void* userData;
+		const E_WRITER_PARAMETER_FLAGS writerFlags;
+		const irr::asset::IAssetLoader::E_LOADER_PARAMETER_FLAGS loaderFlagsUsedForWriting;
```

Comment:
what's `loadderFlagsUSedForWriting` doing here?

Reply:
We have to know the orientation of loaded mesh, I mean the orientation applied to mesh

Reply:
Your proposed approach makes no sense, for two reasons:
1. Not every mesh to be written is loaded (some are generated, i.e. geometry creator)
2. You will use the `writerFlags` to tell the writer if the meshes being passed in are right handed or not (each writer will decide internally whether to flip or not depending on the convention for the format OR if the format can take the orientation metadata like .baw)